src/ngraph_encapsulate_op.cc

```diff
@@ -53,11 +55,6 @@ namespace ng = ngraph;
 
 namespace tensorflow {
```

Comment:
General comment for this file: Move the implementation to it's own file (i.e., `ngraph_encapsulate_impl.cc`.